doc/OnlineDocs/explanation/philosophy/expressions/managing.rst

```diff
@@ -216,19 +202,20 @@ callbacks, which are documented in the class documentation.
 Detailed documentation of the APIs for these methods is provided
 with the class documentation for these visitors.
 
-SimpleExpressionVisitor Example
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+StreamBasedExpressionVisitor Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 In this example, we describe an visitor class that counts the number
 of nodes in an expression (including leaf nodes).  Consider the following
 class:
 
 .. literalinclude:: /src/expr/managing_visitor1.spy
 
-The class constructor creates a counter, and the :func:`visit` method
-increments this counter for every node that is visited.  The :func:`finalize`
-method returns the value of this counter after the tree has been walked.  The
-following function illustrates this use of this visitor class:
+The :func:`initializeWalker` method creates a counter, and the
+:func:`exitNode` method increments this counter for every node that is
+visited.  The :func:`finalizeResult` method returns the value of this
```

Comment:
I don't see any of these methods used in the code snippet rendered in the documentation. Did you make changes locally that were never committed to this branch?